src/display/components/display_bandwidth.rs

```diff
@@ -1,13 +1,13 @@
-use std::fmt;
-
-use derivative::Derivative;
-
 use crate::cli::UnitFamily;
+use derive_more::Debug;
+use std::fmt;
```

Comment:
Just a nit-pick: is this something your formatter did automatically? I think by default RA likes to order the imports in this order: `std`, foreign, `crate`.

Rustfmt doesn't enforce this (at least not yet), but I'd like to stick to RA's preferred formatting just in case. Also it's good for consistency.

Reply:
It might be my formatter, or it might be that I added the import in wrong way and ran cargo fmt manually, which didn't enforce this. In any case, I have updated the order.